src/mcl_3dl.cpp

```diff
@@ -1289,6 +1344,11 @@ class MCL3dlNode
     pnh_.param("publish_tf", publish_tf_, true);
     pnh_.param("output_pcd", output_pcd_, false);
 
+    const float float_max = std::numeric_limits<float>::max();
+    pnh_.param("std_warn_thresh_x", std_warn_thresh_[0], float_max);
+    pnh_.param("std_warn_thresh_y", std_warn_thresh_[1], float_max);
```

Comment:
IMO, having x and y separately doesn't make sense as the error ellipse represented by (x, y) parameter can't represent the major axis orientation of the ellipse.
For mobile robot purpose, separating z may make sense due to the constraint of the floor.

Reply:
@at-wat Addressed your comment.